### ISSUE TITLE
Colorblind character is bugged

### ISSUE THREAD

**Issue by aliblong** [Zamiell/hanabi-live#173]
Tested colorblind in a game. When a **green** clue to the colorblind character was attempted, the error message read "you cannot clue a **greedy** character **blue**". Then a red clue was successfully given to the colorblind character.

**Comment by fpvandoorn** [comment]
The wrong character is addressed in `#170`. The actual text is 
```
s.Warning("You cannot give a color blue to a " + name + " character.")
```
and `clue` is just misspelled `blue`.

**Comment by Zamiell** [comment]
fixed
